src/main.rs

```diff
@@ -37,6 +38,14 @@ fn main() -> Result<()> {
 
     for disk in &disk_parts {
         println!("{} {}", disk.parent_path.display(), disk.partition);
+
+        if options.automatically_grow {
+            println!("Growing the partition and expanding the vdev...");
+            grow::grow(&options.zpool_name, disk, options.dry_run).unwrap();
```

Comment:
Nit: Should we use  the try operator instead of unwrapping, or do we not really care? (It would require implementing `Error` or `From<GrowError>`.)

Reply:
I'm flexible, though I don't know how to convert From<GrowError> to Error. Can you help?